sim/components/battery/BatteryController.h

```diff
@@ -21,6 +21,10 @@ namespace Pinetime {
         return voltage;
       }
 
+      bool BatteryIsLow() const {
+        return false;
```

Comment:
let us let this stub do something. When would the battery considered to be low? let's say about 10 percent (other suggestions are welcome)

```suggestion
        return percentRemaining <= 10;
```

Reply:
The original implementation includes a constant which is 20. I make another commit :grin: